hack/ginkgo-e2e.sh

```diff
@@ -105,8 +109,8 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --cluster-tag="${CLUSTER_ID:-}" \
   --repo-root="${KUBE_VERSION_ROOT}" \
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
-  --num-nodes="${NUM_NODES:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  ${num_nodes_arg} \
```

Comment:
You can do this with:

```
${NUM_NODES:+"--num-nodes=${NUM_NODES}"}
```

Liked `E2E_MIN_STARTUP_PODS` below.

Is it worth just changing all of these (`prefix`, `node-instance-group`, etc.)?